### ISSUE TITLE
Jenkins fails to compile the newer builds - missing dependencies.

### ISSUE THREAD

**Issue by mibby** [jyhsu2000/BannerMaker#7]
@jyhsu2000 Jenkins fails to compile the newer builds.

```
[ERROR] Failed to execute goal on project BannerMaker: Could not resolve dependencies for project tw.kid7:BannerMaker:jar:1.5.0: Failed to collect dependencies for [org.bukkit:bukkit:jar:1.11.2-R0.1-SNAPSHOT (provided), net.milkbowl.vault:VaultAPI:jar:1.6 (provided), org.bstats:bstats-bukkit:jar:1.1 (compile)]: Failed to read artifact descriptor for org.bukkit:bukkit:jar:1.11.2-R0.1-SNAPSHOT: Could not transfer artifact org.bukkit:bukkit:pom:1.11.2-R0.1-SNAPSHOT from/to spigot-repo (https://hub.spigotmc.org/nexus/content/repositories/snapshots/): peer not authenticated -> [Help 1]
```

**Comment by jyhsu2000** [comment]
Thanks for reporting.

Yesterday I found it when it just happened.
After some trying and researching, I think the problem is the SSL connection between my Jenkins server and the spigotmc repo server.
As you see on the Jenkins, I tried several kinds of settings to fix this in these days.
But so far I didn't found the valid solution.

If you want to get the newer version immediately before I fix that,
you can also clone or download this repo and use maven to build it.

**Comment by jyhsu2000** [comment]
Fixed in [build#83](https://ci.kid7.club/job/BannerMaker/83/)

Solution:
Update to Java 8
